lib/error.js

```diff
@@ -327,20 +327,27 @@ AppError.requestBodyTooLarge = function () {
   })
 }
 
-AppError.tooManyRequests = function (retryAfter) {
+AppError.tooManyRequests = function (retryAfter, retryAfterLocalized) {
   if (!retryAfter) {
     retryAfter = 30
   }
+
+  var extraData = {
```

Comment:
I called it `extraData` because the arg is called `extra` in:

```js
function AppError(options, extra, headers) {
  this.message = options.message || DEFAULTS.message
  this.isBoom = true
```

`responseData` seems wrong because `retryAfter*` stuff is not the only data returned. 
Thoughts?